### ISSUE TITLE
Add 'compact density' support

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#396]
In compact density mode, folders and messages are closer together; the menu items that represent folders (all popups triggered by QuickFolders tabs) should also allow for a similar density, based on the new menu font-size (see #394)

![image](https://github.com/RealRaven2000/QuickFolders/assets/6314674/9cfd7340-91ae-49ca-8f4c-ce57f9b120a8)

**Comment by RealRaven2000** [comment]
Test version below. This will only make menus denser that contain folder entries (the tools menus won't be affected). It is based on the menu font size which can be configured on the QuickFolders layout tab.

[QuickFolders-mx-6.2pre44.zip](https://github.com/RealRaven2000/QuickFolders/files/12421183/QuickFolders-mx-6.2pre44.zip)

Magic rules:

```css
:root[uidensity="compact"] .QuickFolders-folder-popup menuitem {
  min-height: 1.25em;
  padding-block: 2px;
}
```

**Comment by RealRaven2000** [comment]
Version for Thunderbird 102:

[QuickFolders-mx-5.18pre14.zip](https://github.com/RealRaven2000/QuickFolders/files/12421411/QuickFolders-mx-5.18pre14.zip)

Restart maybe required to see layout changes (styles sheets are usually cached by Mozilla Browsers)

**Comment by RealRaven2000** [comment]
Implemented in 6.2 - Published 30/08/2023 
also in 5.18 published 27/08/2023
